tests/unit/butterfree/transform/transformations/conftest.py

```diff
@@ -62,7 +62,7 @@ def target_df_spark(spark_context, spark_session):
             "timestamp": "2016-04-11 11:31:11",
             "feature1": 200,
             "feature2": 200,
-            "feature__cos": 0.48718767500700594,
+            "feature__cos": 0.4871876750070059,
```

Comment:
Why this change?

Reply:
The expected value it is without this last number